packages/wallets/src/components/Base/IconButton/IconButton.scss

```diff
@@ -0,0 +1,92 @@
+$color-map: (
+    'primary': (
+        backgroundcolor: #ff444f,
+        hover: #eb3e48,
+    ),
+    'white': (
+        backgroundcolor: #fff,
+        hover: #d6dadb,
+    ),
+    'black': (
+        backgroundcolor: #0e0e0e,
+        hover: #323738,
+    ),
+    'transparent': (
+        backgroundcolor: transparent,
+        hover: #d6dadb,
+    ),
+);
+
+$size-map: (
+    sm: (
+        padding: 0.4rem,
+        height: 2.4rem,
+    ),
+    md: (
+        padding: 0.8rem,
+        height: 3.2rem,
+    ),
+    lg: (
+        padding: 1.2rem,
+        height: 4rem,
+    ),
+);
+
+$border-radius-map: (
+    sm: 0.4rem,
+    md: 6.4rem,
+    lg: 50%,
+);
+
+.wallets-icon-button {
+    display: grid;
+    place-content: center;
+    border: none;
+
+    &:hover {
+        cursor: pointer;
+    }
+
+    &:disabled {
+        display: none;
+    }
+
+    &__border-radius {
+        &--round {
```

Comment:
nitpick: good habit for future consideration is to put modifiers on top of the component - it generates way cleaner css and limits complexity of component